### ISSUE TITLE
Feature request: hide reruns

### ISSUE THREAD

**Issue by Andrey-Sisoyev** [kwaschny/unwanted-twitch#10]
Some streamers use Twitch feature to broadcast VODs. When browsing available broadcasts Twitch shows both "Live" and "Reruns" broadcasts. Some users prefer to never watch "Reruns", and their presence only distracts them, when browsing through available broadcasts.

**Comment by kwaschny** [comment]
Absolutely. Thoughts about the initial state? Hide reruns by default or have the user check it first?

**Comment by Andrey-Sisoyev** [comment]
Probably better have it disabled (do not hide Reruns) by default - that's just how UI is traditionally configured. Some users may get offended if extension hides something from them without them knowing that.
